### DISCUSSION TITLE
TouchedFields not updating as expected - useFIieldArray move()

### DISCUSSION THREAD

**Discussion by keisto** [react-hook-form/react-hook-form#9590] [Q&A] [answered]
## Version Number

7.40.0

## Codesandbox

https://codesandbox.io/s/react-hook-form-v7-controller-forked-usefieldarray-wm7hon?file=/src/index.js:0-1537

## Steps to reproduce

1. Go to https://codesandbox.io/s/react-hook-form-v7-controller-forked-usefieldarray-wm7hon?file=/src/index.js:0-1537 
2. Select left|right buttons to move around A,B,C
3. Click 'submit'
4. Check console for `touchedFields`

## What browsers are you seeing the problem on?

Chrome, Safari, Firefox

## Expected behavior / Question

Although the form is correctly marked dirty, I would expect the `touchedFields` to include `testFieldArray` (controller key passed to useFieldArray)

**Comment by bluebill1049** [comment, marked as answer]
`move` action is not going to update touched fields, touched fields get updated with `onBlur` event.
